### DISCUSSION TITLE
[Feature Request] Global Gaps

### DISCUSSION THREAD

**Discussion by andynameistaken** [asmvik/yabai#1544] [Ideas]
Hi, could you add functionality to `yabai` that repicates this script:

```
#!/bin/bash

args="$*"

while read -r l
do
  yabai -m space "$l" $args
done <<< "$(yabai -m query --spaces  | jq 'map(select(."is-native-fullscreen" == false))' | awk '/index/' | sed 's/[^0-9]*//g')"

```

**Comment by dev-msp** [comment]
You titled the topic "global gaps", but the script you pasted is a generic script that runs `yabai -m space [args]` for every space that isn't a full-screened window. Is something missing from the script?

In any case, it's safe to say that anything you can already do with a simple script is not likely to be a good `yabai` feature candidate.
